nalosman/theeirenecollection
# download from production theme

### Goal
Brief description of what this request seeks to accomplish.  

### Deployment Instructions
Any specific instructions that must be followed before, after, or during deployment.

### Testing Instructions
Visit [Homepage](https://pbzymg9kcdydok2r-31343181961.shopifypreview.com)
-Ensure All files from the production theme have been downloaded

### Code Request Quality Checklist


### Reviewers
@reviewer1  
@reviewer2  

### Mentions
@mention1  
@mention2
